docs/reference/changelog.rst

```diff
@@ -77,6 +77,13 @@ Changelog
 
   For a complete list of commits, see the `X.Y.Z`_ release on GitHub.
 
+3.2.3 (2025-01-23)
```

Comment:
Are you dating these for the tagging date or the scheduled date to release to stable? (I'm trying the latter in snapcraft)

Reply:
In this *particular* case, both :-)

For 3.3.0 date confusion ended up with it having the tagging date, but in future I'll be doing the stable-release date.